<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseDialog</class>
 <widget class="QDialog" name="chooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>61</y>
     <width>61</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QRadioButton" name="radioButton1">
      <property name="text">
       <string>政治</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton2">
      <property name="text">
       <string>英语</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton3">
      <property name="text">
       <string>数学</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton4">
      <property name="text">
       <string>专业课</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
